src/App.js

```diff
@@ -1,24 +1,45 @@
-import logo from './logo.svg';
 import './App.css';
+import './fonts/Hind-Regular.ttf'
+import './fonts/Montserrat-VariableFont_wght.ttf'
+import Header from './components/Header'; 
+import Form from './components/Form';
+import Gallery from './components/Gallery'
+import Footer from './components/Footer';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 function App() {
```

Comment:
can't see your index.html file to comments so leaving this here - remember to change the title of your app so it shows up in your browser tab you can also change your favicon there as well